CombinatorialGames/Game/IGame.lean

```diff
@@ -394,6 +397,9 @@ infix:50 " ≈ " => AntisymmRel (· ≤ ·)
 `IncompRel (⬝ ≤ ⬝) x y`. -/
 notation:50 x:50 " ‖ " y:50 => IncompRel (· ≤ ·) x y
 
+recommended_spelling "equiv" for "≈" in [«term_≈_»]
+recommended_spelling "fuzzy" for "‖" in [«term_‖_»]
```

Comment:
(the spacing for this (and `recommended_spelling "lf" for "⧏" in [«term_⧏_»]`) seems a little bit inconsistent with the other `recommended_spelling` declarations?)